fetch.bs

```diff
@@ -1417,42 +1417,45 @@ Unless stated otherwise, it is unset.
 "<code>no-cache</code>", "<code>force-cache</code>", or
 "<code>only-if-cached</code>". Unless stated otherwise, it is "<code>default</code>".
 
+<p>Let <var>httpCache</var> be the result of <a>determining the HTTP cache partition</a> for
+<a for=/>request</a>. If <var>httpCache</var> is null, fetch behaves as if there is no HTTP cache
+at all and the <a>cache mode</a> is considered "<code>no-cache</code>".
+
 <div class="note no-backref">
  <dl>
   <dt>"<code>default</code>"
-  <dd><a for=/>Fetch</a> will inspect the HTTP cache on the way to the network. If the HTTP cache
-  contains a matching <a>fresh response</a> it will be returned. If the HTTP cache contains a
-  matching <a>stale-while-revalidate response</a> it will be returned, and a conditional network
-  fetch will be made to update the entry in the HTTP cache. If the HTTP cache contains a matching
-  <a>stale response</a>, a conditional network fetch will be returned to update the entry in
-  the HTTP cache. Otherwise, a non-conditional network fetch will be returned to update the entry
-  in the HTTP cache. [[!HTTP]] [[!HTTP-SEMANTICS]] [[!HTTP-COND]] [[!HTTP-CACHING]] [[!HTTP-AUTH]]
-  [[!STALE-WHILE-REVALIDATE]]
+  <dd><a for=/>Fetch</a> will inspect the <var>httpCache</var> on the way to the network. If the
+  <var>httpCache</var> contains a matching <a>fresh response</a> it will be returned. If the
```

Comment:
These non-normative explanations of different cache modes should not refer to a variable. Probably they should be reverted; they were reasonable before.

Reply:
done